### DISCUSSION TITLE
BMW ID4 NBT no BMW Apps showing

### DISCUSSION THREAD

**Discussion by gelno55** [BimmerGestalt/AAIdrive#384] [Q&A]
Hello I've come across this app that was recommended by someone on facebook.
I've installed the My BMW app and also have a active BMW Connected subscription. However there is no "BMW Apps" section under my Connected Drive in the iDrive menu. Am I missing something?

Thank you!

**Comment by hufman** [comment]
Welcome! Let's get this problem figured out. It sounds like you're [looking in the right place](https://github.com/BimmerGestalt/AAIdrive/blob/main/app/src/main/res/drawable/pic_connassistant_bmw.jpg).

I think the BMW Connected subscription should be all that's needed. Did you change the subscription status recently? The car might take a day or two to notice, and there should be a menu entry to force it to update the subscription status immediately. I've also heard that sometimes the car disables that feature if it can't contact the cloud from within the garage at night. I'd of course recommend contacting BMW or the dealer, but they'd probably just say the car is unsupported and to buy a new one 😉 

Unfortunately I have a Mini which doesn't have a subscription service, but hopefully someone else can chime in with helpful tips they used in this same situation!

**Reply by gelno55** [reply]
Thank you for your reply!

Yes I've just purchased the subscription about 4 hours ago so it could be that the car has not recieved the new status yet. I did notice my car does not have the 6NR option when i decode my VIN number. Could that have something to do with it?
I did contact BMW about this and they said if it didn't come from the factory that it would be impossible to add it to the car later.

**Reply by hufman** [reply]
Oh that's excellent news, I've heard of several people who bought the subscription and then the car didn't notice until a day later so it probably will still work.
I think the note about 6NR may be incorrect, I'm starting to suspect it may be the coding option for the previous generation of IDrive that only supports iOS apps. It seems that BMW switched entirely to subscriptions for IDrive4 and later. It may be that the 6NR code option still exists to force the app support even without the subscription, but I'm not familiar enough with coding to say for certain.

**Reply by gelno55** [reply]
That's good to hear there still may be a chance 😄 
I will update you tomorrow or in about 2 days to see if the option is availabe then!

**Reply by matztam** [reply]
6NR can be coded by some third party companies. The costs are around 80€. 

P.S. With the MyBMW app I don't get the apps displayed either. So far it only works for me with the Connected Classic app (version 1.8).

**Reply by gelno55** [reply]
I've waited 2 days and it's still not showing sadly.
Is there any way I can download the apk from somewhere because the classic app is not on the play store anymore

**Reply by matztam** [reply]
You can get it here: https://apkpure.com/de/bmw-connected-classic/com.bmwgroup.connected.bmw/versions

**Reply by gelno55** [reply]
I've tried the legacy app but it says it can't connect to the car. I guess I'll have to find someone to code the 6NR option in my car.

**Reply by hufman** [reply]
I apologize, and I wish it could work more smoothly :(
